packages/kbn-esql-validation-autocomplete/src/definitions/generated/scalar_functions.ts

```diff
@@ -5367,7 +5541,6 @@ const mvSortDefinition: FunctionDefinition = {
           name: 'order',
           type: 'keyword',
           optional: true,
-          acceptedValues: ['asc', 'desc'],
```

Comment:
I think we need to put this back cause otherwise the mv_sort will stop giving errors for non asc or desc for secon param.

Reply:
They got removed from the script, are we adding them manually?

Reply:
You were right! We do this manually, great catch! https://github.com/elastic/kibana/pull/201078/commits/9871adf989340dc5cc76c9a86271c21c0fc344d2